.github/workflows/run-tests.yml

```diff
@@ -22,76 +22,99 @@ on:
 
 jobs:
 
-  _:
-    runs-on: ubuntu-latest
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+          include:
+            - os: ubuntu-latest
+              name: linux
+              dotnet_script: ./dotnet.sh
+              build_script: ./build.sh
+            - os: windows-latest
+              name: windows
+              dotnet_script: .\dotnet.cmd
+              build_script: .\build.cmd
     timeout-minutes: 60
-    name: ${{ inputs.testShortName }}
+    name: ${{ matrix.name }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
 
       - name: Trust HTTPS development certificate
-        run: ./dotnet.sh dev-certs https --trust
+        if: matrix.os == 'ubuntu-latest'
+        run: ${{ matrix.dotnet_script }} dev-certs https --trust
```

Comment:
Why don't the dev-certs need to be trusted on Windows?

Reply:
I wasn't able to get that running without prompt popping up. I seem to recall that you can pass `echo yes` or something to it, but couldn't get something working. And .. the tests seem to be passing anyway? So, I have this disabled right now.